skeleton/aws/modules/alb/main.tf

```diff
@@ -1,4 +1,8 @@
 # tfsec:ignore:aws-elb-alb-not-public
+locals {
+  enable_stickiness = false
+}
+
```

Comment:
I think the `locals` should be put above the `#tfsec:ignore` as it belongs to the bellow resource.
```suggestion
locals {
  enable_stickiness = false
}

# tfsec:ignore:aws-elb-alb-not-public
```

Reply:
Nice catch!! I'm working too much with Ruby and the frozen_literal_string :harold:

19e0b11 ✅